manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```
@@ -87,6 +87,9 @@ spec:
             httpdCpuRequest:
               description: 'Httpd deployment CPU request (default: no request)'
               type: string
+            httpdImage:
+              description: 'Image string used for the httpd deployment (default: <HttpdImageNamespace>:<HttpdImageTag>)'
```

This one is slightly wrong too...

```suggestion
              description: 'Image string used for the httpd deployment (default: <HttpdImageNamespace>/httpd:<HttpdImageTag>)'
```

or perhaps

```suggestion
              description: 'Image string used for the httpd deployment (default: <HttpdImageNamespace>/httpd[-init]:<HttpdImageTag>)'
```